packages/app-content-pages/src/screens/Publications/Publications.js

```diff
@@ -55,21 +50,22 @@ function Publications ({
   )
 
   const sidebar = (
-    <Box as='ul' gap='small'>
+    <Box as='menu' gap='small' name='Filter by category'>
       {filters.map(filter => (
-        <StyledLi key={filter.name}>
-          <Link
-            href={filter.slug ? `#${filter.slug}` : ''}
-            passHref
-          >
-            <StyledButton
-              active={filter.active}
-              label={filter.name}
-              onClick={filter.setActive}
-              plain
-            />
-          </Link>
-        </StyledLi>
+        <Link
+          href={filter.slug ? `#${filter.slug}` : ''}
+          key={filter.name}
+          passHref
+          role='menuitemcheckbox'
```

Comment:
I’d remove this role too, since the link here is being used as a link so we don’t want to override its role.

Reply:
These links also need to be wrapped in list items, in order for this to be valid HTML.

Reply:
Seems like this PR is reverting some of the accessibility fixes from #3599.

Reply:
For clarity, do the links need to be wrapped in list items because their parent is `ul`? Or do they need to be wrapped in list items simply because they're links?

Reply:
With the code here, it’s because the parent is [`<menu>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu), but the same rule is true for `<ul>` too. The children of a list element must be list items.

Reply:
Here are some examples:
```html
<!-- valid list of links -->
<ul>
  <li><a href="#astronomy">Astronomy</a></li>
  <li><a href="#nature">Nature</a></li>
</ul>

<!-- valid menu (browsers parse <menu> as <ul>) -->
<menu>
  <li><a href="#astronomy">Astronomy</a></li>
  <li><a href="#nature">Nature</a></li>
</menu>

<!-- invalid menu (<a> can't be a direct child of <menu>) -->
<menu>
  <a href="#astronomy">Astronomy</a>
  <a href="#nature">Nature</a>
</menu>
```

Reply:
Thanks! I was a bit confused with this feature because it's a menu of links, but its UX is a filter feature.

Reply:
With the previous implementation, which was a list of buttons (no links), it wasn’t clear that a blind reader would know the main content had changed after they pressed a button. 

Here, clicking a link navigates you to the corresponding heading, which will be announced.

Reply:
I guess this pattern has the added bonus that you don’t lose your place if you refresh the page.